models/vision/detection/yolo/Makefile

```diff
@@ -7,6 +7,7 @@ SHELL := /bin/bash
 
 flags=-O3
 cv_libs:=-L/opt/opencv/3.4.11/lib64 -lopencv_core -lopencv_imgcodecs -lopencv_imgproc -lopencv_videoio
+LDLIBS = -lpopart -lpoplar -lpopops
```

Comment:
I think we don't need to link these libs. libodla_popart.so will link against these libs.